waterloop-site/src/static/ste/icons/cloud.svg

```diff
@@ -0,0 +1 @@
.06.1.11.16.128.22.011.03-.06.069-.099.11Zm93.471 20.171c.05.049.121.078.151.138.009.032-.041.091-.071.14l-.199-.129c.05-.06.089-.1.119-.149Zm-30.187.388c.078-.08.149-.159.229-.25.039.041.069.08.108.121-.089.07-.179.138-.268.209-.03-.03-.051-.06-.069-.08Zm-34.966-21.068c.049.05.099.099.129.158.009.012-.08.11-.099.101-.06-.02-.11-.08-.179-.13.078-.06.119-.1.149-.129Zm-7.833-.527c.018-.012.108.078.099.098-.03.06-.081.11-.129.17-.03-.03-.071-.06-.119-.11l-.06.45c-.062-.012-.13-.021-.19-.03.03-.17.06-.33.089-.5l.17.03c.05-.04.089-.079.14-.108Zm-22.117 18.17c-.06.059-.1.107-.15.158l-.128-.129c.048-.05.099-.1.158-.14.01-.02.06.049.12.11Zm120.942.456c-.039.06-.099.13-.159.14-.05 0-.098-.08-.199-.17.119-.04.19-.09.25-.078.039 0 .119.099.108.108Zm.995.02.091-.03c.04.09.09.18.129.26-.05.009-.099.03-.149.039-.021-.09-.041-.18-.071-.269Zm-2.447-.069c.06.03.119.079.138.129.011.03-.039.09-.078.179-.101-.129-.14-.179-.19-.239.059-.02.109-.08.13-.069Zm-90.934-15.799c.009.02-.08.11-.089.1-.06-.03-.11-.08-.191-.14.071-.049.14-.12.151-.109.06.04.108.09.129.15Zm63.96 16.077c-.039-.06-.089-.12-.11-.179-.009-.02.071-.069.14-.14.03.101.069.17.069.23.011.04-.06.06-.099.09Zm10.888-.01c.05-.038.101-.089.151-.119.019-.01.069.07.119.12-.059.03-.11.08-.149.08-.041.01-.08-.05-.121-.08Zm-85.676.21c.069.149-.11.3-.23.449l.06.06-.13.098c-.07-.099-.138-.188-.21-.289.03-.018.06-.05.102-.069.03.03.06.07.098.099-.01-.179-.02-.369-.03-.596.1.039.17.078.23.108a.23.23 0 0 1 .078-.09c.02-.009.091.041.14.071-.049.06-.07.108-.108.159Zm86.572.389.06-.24c.05.02.09.04.14.05l-.119.209c-.03-.01-.051-.01-.081-.019ZM94.518 18.986c.099-.04.159-.08.218-.09.03 0 .06.071.09.101-.06.04-.108.09-.168.12-.02.009-.062-.06-.14-.131Zm11.635.209a3.512 3.512 0 0 0-.19-.12c.051-.069.09-.108.12-.149.05.041.119.08.149.14.011.02-.049.09-.079.129ZM94.747 36.2c0-.08.04-.15.09-.299.128.21.179.278.167.29a.329.329 0 0 1-.167.107c-.02.011-.09-.069-.09-.099Zm-7.117 0c-.05.069-.1.149-.14.22-.049-.05-.09-.11-.138-.16.09-.03.188-.04.278-.06Zm.259.637c.009.02-.08.11-.09.099-.06-.03-.11-.079-.19-.15.071-.059.14-.119.151-.11.049.051.099.101.129.16Zm-.975-18.408c0 .02-.08.06-.12.08-.04-.06-.09-.11-.1-.18-.01-.05.02-.109.03-.16.04.021.08.051.12.07.02.06.059.119.07.19Zm-2.747 18.717c-.03 0-.06-.01-.08-.01v-.368c.039.01.08.01.119.019-.01.06-.01.13-.02.19.04-.02.089-.05.13-.071.069.13.129.259.2.39-.051.03-.101.05-.152.08-.06-.11-.119-.23-.188-.34-.01.03-.01.07-.01.11Zm4.02-.36c.05 0 .12.03.14.071.009.03-.04.129-.06.129-.069 0-.13-.04-.25-.08.09-.07.13-.12.17-.12Zm64.976 5.191c0-.06.05-.12.089-.23.069.081.14.13.129.15-.019.06-.079.12-.12.17-.039-.03-.108-.06-.098-.09Zm-17.13.11c-.03-.04-.06-.09-.119-.18.05-.03.098-.079.149-.07.039.01.099.08.089.11-.02.05-.08.09-.119.14Zm-35.901-27.634-.09.181a2.516 2.516 0 0 0-.19-.08c.041-.06.08-.12.13-.16.03-.019.099.04.15.06ZM86.206 41.798c.03 0 .11.07.1.09-.01.06-.06.099-.119.188-.07-.099-.12-.15-.16-.209.07-.02.12-.069.179-.069Zm-8.829.019c.02-.01.09.04.13.06-.03.061-.059.13-.1.18-.01.01-.09-
```

Comment:
Can you pass this SVG markup, as well as the other ones, through [SVGOMG](https://jakearchibald.github.io/svgomg/) to clean it up a bit and also prettify?

Reply:
Also, could you move all SVGs too `waterloop-website-w20/waterloop-site/src/static/img/sustainable-tech`?